<?xml version="1.0"?>
<launch>
<arg name="ros_port" default="0"/>
<arg name="gaz_port" default="0"/>
<arg name="ros_ip" default="0"/>
<param name="use_sim_time" value="false"/>
<arg name="drone_name" default="copter"/>
  <group ns="$(arg drone_name)">
    <node name="vicon_gym_node" pkg="training_q_learning" type="vicon_test_model_2D.py" output="screen" >
        <param name = "drone_name" value = "$(arg drone_name)"  />
    </node> 
  </group>
</launch>